### DISCUSSION TITLE
Media query styles not being applied.. !important classes are taking priority.

### DISCUSSION THREAD

**Discussion by mforcer** [tailwindlabs/tailwindcss#5099] [Help]
This may well be a bug, but I thought I'd post here first.

I've noticed on some elements that any padding/margin applied to media queries is being ignored.. When inspecting the classes and CSS applied in the Chrome dev tools I can see that the mobile classes are being applied twice.. and !important is being applied in one of them.

This is strange because I am not using !important in Tailwind, I never have... I can see this can be toggled in the config file, it's not in my file at all.

on my element my CSS attributes are: `mt-2 ml-2 md:m-0`

after `md` I'd expect the margin to be set to 0 but it's not working. 

Here is the output:

<img width="246" alt="Screenshot 2021-07-28 at 00 47 50" src="https://user-images.githubusercontent.com/850149/127241743-acbb4944-3bd5-44fc-8bff-b5bda48f02fb.png">

`md:m-0` is never applied... why is ml-2/mt-2 being applied twice?

I tried adding !important to it in dev tools, then it works.

I found another person with the same issue on Laracasts: https://laracasts.com/discuss/channels/general-discussion/tailwindcss-mobile-first

(I can see he has messed up padding and margin classes in his first post, this is obviously a mistake in the post, not the code, if you read on he goes on to describe the same issue).

**Comment by simonswiss** [comment]
Are you able to reprroduce this in a little GitHub repo?

What you're describing definitely should work (and it does here: https://play.tailwindcss.com/odpC6xU9Qa), so there's likely something wrong in your setup, and it's. impossible to tell without seeing some code.
